Docs/moderation: add mod-command list (#7)
AyuAi/beemo-docs: moderation/moderation.md

Beemo offers a wide range of moderation tools to help handle any problematic users. Explore the tabs below to see how.
{: .fs-6 .fw-300 }

## Moderation Commands

| Command       | Example                                     | Usage                                                                                                                                                                                                                                                              |
| ------------- | ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | Archive       | `/archive [ID/Mention]`                     | Creates a temporary message archive for the user specified                                                                                                                                                                                                         |

Reviewer: ```suggestion
| Archive       | `/archive [User ID/Mention]`                     | Creates a message archive, which expires in 12 hours later, for the user specified. Archives cannot be generated for moderators or admins to ensure messages in private channels stay protected.                                                                                                                                                                                                         |
```

> | Avatar        | `/avatar [ID/Mention]`                      | Gets the avatar of the specified user                                                                                                                                                                                                                              |

Reviewer: ```suggestion
| Avatar        | `/avatar [User ID/Mention]`                      | Gets the avatar of the specified user.                                                                                                                                                                                                                              |
```

> | Ban           | `/ban [ID/Mention] (Duration) (Reason)`     | Bans the user, deleting their past 7 days of messages and notifying them through DMs. This incident is also logged. Not specifying a duration sets the ban to permanent                                                                                            |

Reviewer: ```suggestion
| Ban           | `/ban [User ID/Mention] (Duration) (Reason)`     | Bans the user, deleting their past 7 days of messages and notifying them through DMs. This incident is also logged. Not specifying a duration sets the ban to permanent.                                                                                            |
```

> | Cleankick     | `/cleankick [ID/Mention] (Reason)`          | Kicks the user and deletes their past 7 days of messages, notifying them through DMs and logging the incident                                                                                                                                                      |

Reviewer: ```suggestion
| Cleankick     | `/cleankick [User ID/Mention] (Reason)`          | Kicks the user and deletes their past 7 days of messages, notifying them through DMs and logging the incident.                                                                                                                                                      |
```

| Duration      | `/duration [Case Num] [New Duration]`       | Edits the duration of an existing moderation case. This will also edit the DM for the user who received the action to reflect the change. You may only edit your own cases unless you are a moderation manager                                                     |

> | Kick          | `/kick [ID/Mention] (Reason)`               | Kicks the user, notifying them through DMs and logging the incident                                                                                                                                                                                                |

Reviewer: ```suggestion
| Kick          | `/kick [User ID/Mention] (Reason)`               | Kicks the user, notifying them through DMs and logging the incident.                                                                                                                                                                                                |
```

> | Modlogs       | `/modlogs [ID/Mention]`                     | Shows all moderation logs of the user.                                                                                                                                                                                                                             |

Reviewer: ```suggestion
| Modlogs       | `/modlogs [User ID/Mention]`                     | Shows all moderation logs for a user, including a link to each cases' action log and message archive.                                                                                                                                                                                                                             |
```

> | Modnote       | `/modnote [ID/Mention] [Reason]`            | Adds a mod note onto the user's logs; This note is only visible to moderators, and does not DM the user or show when they types /mylogs                                                                                                                            |

Reviewer: ```suggestion
| Modnote       | `/modnote [User ID/Mention] [Reason]`            | Adds a mod note onto the user's logs; This note is only visible to moderators, and does not DM the user or show when the user types `/mylogs`.                                                                                                                            |
```

> | Modstats      | `/modstats [ID/Mention]`                    | Gets statistics of the specified moderator. You must be a moderation manager or admin to use this command unless you are invoking it on yourself                                                                                                                   |

Reviewer: ```suggestion
| Modstats      | `/modstats [Moderator ID/Mention] (before:YYYY/MM/DD) (after:YYYY/MM/DD)`                    | Gets statistics of the specified moderator. You can optionally specify a before and/or after date to get stats for specific time periods. You must be a moderation manager or admin to use this command.                                                                                                                   |
```

> | Mute          | `/mute [ID/Mention] (Duration) (Reason)`    | Mutes the user, notifying them through DMs when the mute begins and ends and logging the incident. Not specifying a duration sets the mute to permanent                                                                                                            |

Reviewer: ```suggestion
| Mute          | `/mute [User ID/Mention] (Duration) (Reason)`    | Mutes the user, notifying them through DMs when the mute begins and ends and logging the incident. Not specifying a duration sets the mute to permanent.                                                                                                            |
```

> | Presetreasons | `/presetreasons (preset)`                   | Lists all available preset reason variables for the server. If a preset variable is specified, this command will list the correlated preset reason                                                                                                                 |

Reviewer: ```suggestion
| Presetreasons | `/presetreasons (preset)`                   | Lists all available preset reason variables for the server. If a preset variable is specified, this command will list the correlated preset reason.                                                                                                                 |
```

> | Purge         | `/purge [Amount <= 1000]`                   | Mass-deletes the specified number of messages in the channel invoked. This command has a limit of 1000 messages, and may take a few seconds. You must be a moderation manager or admin to use this command                                                         |

Reviewer: ```suggestion
| Purge         | `/purge [Amount <= 1000]`                   | Mass-deletes the specified number of messages in the channel invoked. This command has a limit of 1000 messages, and may take a few seconds. You must be a moderation manager or admin to use this command.                                                         |
```

> | Reason        | `/reason [Case Num] [New Reason]`           | Edits the reason of an existing moderation case. This will also edit the DM for the user who received the action to reflect the change. You may only edit your own cases unless you are a moderation manager                                                       |

Reviewer: ```suggestion
| Reason        | `/reason [Case Num] [New Reason]`           | Edits the reason of an existing moderation case. This will also edit the DM for the user who received the action to reflect the change. You may only edit your own cases unless you are a moderation manager or admin.                                                       |
```

> | Revoke        | `/revoke [Case Num] (Reason)`               | Revokes the specified moderation case. This will prevent the case from showing on /modlogs and will edit the DM for the user who received the action to reflect that the action is revoked. You may only revoke your own cases unless you are a moderation manager |

Reviewer: ```suggestion
| Revoke        | `/revoke [Case Num] (Reason)`               | Revokes the specified moderation case. This will prevent the case from showing on /modlogs and will edit the DM for the user who received the action to reflect that the action is revoked. You may only revoke your own cases unless you are a moderation manager or admin. |
```

> | Revokeall     | `/revokeall [Case Num] (Reason)`            | Revokes all cases tied to a user, preventing them from showing up on their moderation logs. You must be a moderation manager or admin to use this command                                                                                                          |

Reviewer: ```suggestion
| Revokeall     | `/revokeall [User ID/Mention] (Reason)`            | Revokes all cases tied to a user, preventing them from showing up on their moderation logs. You must be a moderation manager or admin to use this command.                                                                                                          |
```

> | Saveban       | `/saveban [ID/Mention] (Duration) (Reason)` | The same as /ban except the user’s messages are not deleted                                                                                                                                                                                                        |

Reviewer: ```suggestion
| Saveban       | `/saveban [User ID/Mention] (Duration) (Reason)` | The same as `/ban`, except the user’s messages are not deleted.                                                                                                                                                                                                        |
```

> | Unban         | `/unban [ID/Mention] (Reason)`              | Unbans the user, attempting to DM them and logging the incident                                                                                                                                                                                                    |

Reviewer: ```suggestion
| Unban         | `/unban [User ID/Mention] (Reason)`              | Unbans the user, attempting to DM them and logging the incident.                                                                                                                                                                                                    |
```

> | Unmute        | `/unmute [ID/Mention] (Reason)`             | Unmutes the user, notifying them through DMs and logging the incident                                                                                                                                                                                              |

Reviewer: ```suggestion
| Unmute        | `/unmute [User ID/Mention] (Reason)`             | Unmutes the user, notifying them through DMs and logging the incident.                                                                                                                                                                                              |
```

> | Userinfo      | `/userinfo [ID/Mention]`                    | Gets information regarding a user in the server, such as account age, join date, join position, and roles                                                                                                                                                          |

Reviewer: ```suggestion
| Userinfo      | `/userinfo [User ID/Mention]`                    | Gets information regarding a user in the server, such as account age, join date, join position, and roles.                                                                                                                                                          |
```

> | Warn          | `/warn [ID/Mention] [Reason]`               | Warns the user, notifying them through DMs and logging the incident                                                                                                                                                                                                |

Reviewer: ```suggestion
| Warn          | `/warn [User ID/Mention] [Reason]`               | Warns the user, notifying them through DMs and logging the incident.                                                                                                                                                                                                |
```
